frontend/src/components/routes/MyRoutes.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {useState} from 'react';
+import Header from "../header/Header";
+import {Route, Routes} from "react-router-dom";
+import Profile from "../../pages/main/profile/Profile";
+import TopUp from "../../pages/main/topup/TopUp";
+import Transactions from "../../pages/main/transactions/Transactions";
+import SignIn from "../../pages/auth/signin/SignIn";
+
+const MyRoutes: React.FC = () => {
+
+    const [isLogged, setIsLogged] = useState(true);
+
+    const RouteMain = (
+        <Routes>
+            <Route path="/profile" element={<Profile />}>
+            </Route>
+            <Route path="/topUp" element={<TopUp />}>
+            </Route>
+            <Route path="/transactions" element={ <Transactions />}>
+            </Route>
+        </Routes>
+    );
+
+    const RouteLogin = (
```

Comment:
Keep consistency across the project. Choose Login or SignIn. Don't mess it every time